features/steps/steps.py

```diff
@@ -523,7 +523,10 @@ def then_stream_does_not_match_regexp(context, stream):
 @then("{stream} matches regexp")
 def then_stream_matches_regexp(context, stream):
     content = getattr(context.process, stream).strip()
-    assert_that(content, matches_regexp(context.text))
+    text = context.text
+    if "<ci-proxy-ip>" in text and "proxy" in context.instances:
```

Comment:
@orndorffgrant @renanrodrigo @lucasmoura Is this okay or should I use regex here https://github.com/CalvoM/ubuntu-advantage-client/blob/eda4b48b5f3adf19cf81e1713aeda09cfaf9f18c/features/proxy_config.feature#L990?

Reply:
I think this is good as it is